scripts/developer_tools/i_dependabot_auto_merge.py

```
@@ -0,0 +1,213 @@
+
+
+def run_gh(args: list[str]) -> subprocess.CompletedProcess[str]:
+    """Run a `gh` CLI command scoped to REPO_OWNER/REPO_NAME. Never raises.
+
+    Retries transient failures (network blips, GraphQL timeouts) up to
+    GH_RETRY_ATTEMPTS times with a linear backoff before returning the last
+    failing result to the caller.
+    """
+    result = None
+    for attempt in range(1, GH_RETRY_ATTEMPTS + 1):
+        result = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)